codeful-logicapp-sample/test-app/.funcignore

```diff
@@ -0,0 +1 @@
+workflow-designtime/
```

Comment:
Can we structure the folders as following in this PR

\codeful-workflows
codeful-workflows\samples
codeful-workflows\samples\recommend-coffee

Reply:
I would also rename every folder and file that does not have a semantically meaningful name e.g. test-app or stateful1